src/main/java/OutputStringHandler.java

```diff
@@ -0,0 +1,52 @@
+// Output Text Formatter for Yapper
+public class OutputStringHandler {
+    // For Command: Add
+    public static void printAddedTask(int taskCount, Task task) {
+        System.out.println(StringStorage.LINE_DIVIDER);
+        System.out.println("Now, ya gotta do this too:");
+        System.out.println( "  " + task.taskToString() );
+        System.out.println("Your list is now THIS BIG: " + taskCount);
+        System.out.println(StringStorage.LINE_DIVIDER);
+    }
+    // For Command: Todo
+    public static void printAddedTask(int taskCount, TaskTodo todo) {
+        System.out.println(StringStorage.LINE_DIVIDER);
+        System.out.println("Now, ya gotta do this too:");
+        System.out.println( todo.taskToString() );
+        System.out.println("Your list is now THIS BIG: " + taskCount);
+        System.out.println(StringStorage.LINE_DIVIDER);
+    }
+    // For Command: Deadline
+    public static void printAddedTask(int taskCount, TaskDeadline taskDeadline) {
+        System.out.println(StringStorage.LINE_DIVIDER);
+        System.out.println("Now, ya gotta do this too:");
+        System.out.println( taskDeadline.taskToString() );
+        System.out.println("Your list is now THIS BIG: " + taskCount);
+        System.out.println(StringStorage.LINE_DIVIDER);
+    }
+    // For Command: Event
+    public static void printAddedTask(int taskCount, TaskEvent taskEvent) {
+        System.out.println(StringStorage.LINE_DIVIDER);
+        System.out.println("Now, ya gotta do this too:");
+        System.out.println( taskEvent.taskToString() );
+        System.out.println("Your list is now THIS BIG: " + taskCount);
+        System.out.println(StringStorage.LINE_DIVIDER);
```

Comment:
An alternative design would be to move the responsibility of outputting task-specific messages into the respective task classes (Task, TaskTodo, TaskDeadline, TaskEvent). This would also ensure that the class definition is not too long.